plugin.video.newyankeeworkshop/addon.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.newyankeeworkshop" name="The New Yankee Workshop" version="0.0.2" provider-name="th0ma5w">
+<addon id="plugin.video.newyankeeworkshop" name="The New Yankee Workshop" version="0.0.3" provider-name="th0ma5w">
 	<requires>
-		<import addon="xbmc.python" version="2.25.0" />
+		<import addon="xbmc.python" version="3.0.0"/>
```

Comment:
Can't do this. Addons with xbmc.python 3.0.0 should be submitted to the matrix branch